app/assets/stylesheets/components/homepage-join-bip.scss

```diff
@@ -0,0 +1,39 @@
+@include b(homepage-join-bip) {
+  @include outer-container;
+  background-image: image-url("join-bip-background.png");
+  margin-bottom: rem-calc(78);
+  padding: rem-calc(97 72);
+  width: $main__width;
+
+  @include e(title) {
+    font-size: rem-calc(32);
+    color: white;
+    font-family: Merriweather;
+    margin-bottom: rem-calc(24);
+  }
+  
+  @include e(text) {
+    color: white;
+    font-size: rem-calc(18);
+    margin-bottom: rem-calc(24 + 11.5);
```

Comment:
As @nym1k requested, I've switched over to direct REM values rather than pixel valaues passed through `rem-calc`. 

The 11.5px seen here was supposed to represent the padding-top of the button below the text. 24px from the exterior of the button, and then 11.5/12 from the text within.

Now that I'm not using pixels though, I've just put this into a single value of 2.219rem.

Some of the direct REM values in the file are a bit overspecified, but I don't think that presents an issue really. I just ran all of the px values taken from the designs and ran them through a calculator. If we really need to reduce the exactness of these values, I can round them to a decimal place, let me know what you would prefer :)